### PR TITLE
CF Release

### PR DESCRIPTION
**Pull request by CFPBot**
### Changed
- **cf-forms:** [patch] Convert fixed pixel units to ems.
- **cf-layout:** [patch] Fix the background size of overlay hero images

## Review

- @cfpb/front-end-team-admin

## I am a bot

After this PR is merged, I will instruct Travis to [automagically](https://github.com/cfpb/capital-framework/tree/master/scripts/npm/prepublish) perform the following steps:

1. Increment Capital Framework's [version](https://github.com/cfpb/capital-framework/blob/canary/package.json#L3) per our [guidelines](https://github.com/cfpb/capital-framework/issues/179).
2. Add a timestamped entry to the [changelog](https://github.com/cfpb/capital-framework/blob/canary/CHANGELOG.md) with the new version and its changes.
3. [Tag](https://github.com/cfpb/capital-framework/tags) the release and push it to GitHub.
4. Publish both [capital-framework](https://www.npmjs.com/package/capital-framework) and any individually updated [components](http://cfpb.github.io/capital-framework/components/) to npm.
5. Update both [canary](https://github.com/cfpb/capital-framework/tree/canary), our development branch, and [master](https://github.com/cfpb/capital-framework/tree/master), our release branch.

If I do something wrong, [blame a human](https://github.com/cfpb/hubot-capital-framework/issues/new).

kitten gif

### CODE REVIEW

File: src/cf-forms/src/molecules/form-fields.less [hunk 1]
```diff
@@ -41,8 +41,8 @@
             .u-visually-hidden();
 
             .lt-ie9 & {
-                height: 20px;
-                width: 20px;
+                height: unit( 20px / @base-font-size-px, em );;
+                width: unit( 20px / @base-font-size-px, em );;
```
Comment (contolini) [inline src/cf-forms/src/molecules/form-fields.less]: Erroneous double semicolons here.

Comment (anselmbradford) [inline src/cf-forms/src/molecules/form-fields.less]: Fix0rd

**Comment by jimmynotjim** [comment]
Looks good now, but I'll let @contolini have the final sign off.

**Comment by Scotchester** [comment]
@anselmbradford and @contolini Make sure to merge this before any other currently-open PR in this repo. I'm not sure we want to find out how to recover from PRs merged while a release PR is currently open.

**Pull request merged**